contrib/start_lint

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+ls -la /obs/src/api
```

Comment:
:thought_balloon: What is this needed for? Maybe a leftover?

Reply:
debugging, it is still failing